src/main/java/io/leonis/torch/component/form/FormBuilder.java

```diff
@@ -0,0 +1,121 @@
+package io.leonis.torch.component.form;
+
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.stream.Stream;
+
+/**
+ * The Interface FormBuilder.
+ *
+ * @author Jeroen de Jong
+ */
+public interface FormBuilder {
```

Comment:
This might be refactored to be based on decorators and use overloading to prevent the types from being lost (and reflections being used). I'll do this separately.